src/Stripe.net/Services/_common/RequestOptions.cs

```diff
@@ -38,5 +38,10 @@ public class RequestOptions
         /// is only used for creating ephemeral keys, which require a specific API version.
         /// </remarks>
         internal string StripeVersion { get; set; }
+
+        public object Clone()
```

Comment:
- I don't think this method needs to be available to our users. We should be able to switch it from being public to it `internal`

Reply:
those changes shouldn't require any changes to your unit test(s)